certbot-ci/certbot_integration_tests/utils/misc.py

```diff
@@ -209,13 +209,14 @@ def manual_http_hooks(http_server_root, http_port):
         shutil.rmtree(tempdir)
 
 
-def get_certbot_version():
+def get_certbot_version(workspace):
```

Comment:
Initially this logic was in a dedicated function because it was used in several places. But it is not the case anymore.

Also as a public method, passing a workspace to get the certbot version does not have an obvious meaning. 

So, what do you think about moving this logic right into the `certbot_call` module, since it is used only there?